### PR TITLE
Improve intro

### PR DESCRIPTION
**Pull request by dalonsoa**
Adds a more detailed course introduction, to set the scene and manage the expectations. Instructors should briefly cover this introductory information at the beginning of the course.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -4,13 +4,54 @@ root: .  # Is the only page that doesn't follow the pattern /:path/index.html
 permalink: index.html  # Is the only page that doesn't follow the pattern /:path/index.html
 ---
 
-This course introduces intermediate concepts and functionality of the version control system Git and the code repository GitHub,
-focused on ensuring an effective and healthy collaborative process.
+This course introduces intermediate concepts and functionality of the version control
+system Git and the code repository GitHub, focused on ensuring an effective and healthy
```
Comment (dandavies99) [inline index.md:8]: phrasing suggestion

```suggestion
system Git and the code repository GitHub. The course is focused on ensuring an effective and healthy
```

File: index.md [hunk 2]
```diff
@@ -4,13 +4,54 @@ root: .  # Is the only page that doesn't follow the pattern /:path/index.html
...
+- Intermediate Git concepts, commands, and terminology, such as branches, rebase, merge and stash, among others.
+- Using GitHub’s Pull Requests to manage contributions from collaborators
+- Using GitHub’s Forks to contribute to other people’s code
+- Using continuous integration to make sure every collaborator's contribution meets an agreed standard
+- Creating releases and adding tags to communicate effectively to users/collaborators what has changed between versions of the code.
+
+## Learning outcomes
+
+After completing this workshop, you will be better able to
+1.	Use advance Git commands to manage the parallel development of multiple features
```
Comment (dandavies99) [inline index.md:34]:
```suggestion
1.	Use advanced Git commands to manage the parallel development of multiple features
```

File: index.md [hunk 3]
```diff
@@ -4,13 +4,54 @@ root: .  # Is the only page that doesn't follow the pattern /:path/index.html
...
+2.	Collaborate effectively with others on a code base to achieve larger goals
+3.	Apply continuous integration to automate tasks
+
+## Delivery of the course
+
+Material will be delivered as a lecture with task following the Carpentries teaching
+style.
+
+- The instructor will walk you through the theoretical material of the course,
+  executing themselves the relevant code or instructions. **You are highly encouraged to
```
Comment (dandavies99) [inline index.md:44]:
```suggestion
 demonstrating the execution of the relevant code and instructions. **You are highly encouraged to
```

File: index.md [hunk 4]
```diff
@@ -4,13 +4,54 @@ root: .  # Is the only page that doesn't follow the pattern /:path/index.html
...
+  code along** and execute the instructions at the same time.
+- Throughout the lessons, there are **yellow boxes** highlighting particularly difficult
```
Comment (dandavies99) [inline index.md:46]:
```suggestion
- Throughout the lessons, there are **yellow boxes** highlighting particularly challenging
```

File: index.md [hunk 5]
```diff
@@ -4,13 +4,54 @@ root: .  # Is the only page that doesn't follow the pattern /:path/index.html
...
+  or important ideas.
```
Comment (dandavies99) [inline index.md:47]:
```suggestion
  or important concepts.
```

File: index.md [hunk 6]
```diff
@@ -4,13 +4,54 @@ root: .  # Is the only page that doesn't follow the pattern /:path/index.html
...
+- There are also exercises in **orange boxes**. The instructor will give you time to try
+  to do them yourselves before going through the solution. This is often available in a
```
Comment (dandavies99) [inline index.md:49]:
```suggestion
  to do them yourself before going through the solution. This is often available in a
```

File: index.md [hunk 7]
```diff
@@ -4,13 +4,54 @@ root: .  # Is the only page that doesn't follow the pattern /:path/index.html
...
+  folded part of the orange box, so you can check it at any time.
+- When doing exercises, put a green sticker in your computer whenever you are done, or a
+  pick/orange one if you need support. A helper will go to you.
```
Comment (dandavies99) [inline index.md:52]: pink?
```suggestion
  pink/orange one if you need support. A helper will go to you.
```

File: index.md [hunk 8]
```diff
@@ -4,13 +4,54 @@ root: .  # Is the only page that doesn't follow the pattern /:path/index.html
...
+- For online sessions, rise your hand if you are done with the exercise and write
```
Comment (dandavies99) [inline index.md:53]:
```suggestion
- For online sessions, raise your hand if you are done with the exercise and write
```

File: index.md [hunk 9]
```diff
@@ -4,13 +4,54 @@ root: .  # Is the only page that doesn't follow the pattern /:path/index.html
...
+  directly your question or problem in the chat, so a helper can try to solve it.
```
Comment (dandavies99) [inline index.md:54]:
```suggestion
 any questions or problems directly into the chat, so a helper can try to solve it.
```

**Review by dandavies99 — CHANGES_REQUESTED** [review]
Looks thorough and gives a nice overview of the course. Just a phew minor, mostly phrasing suggestions from me.

**Review by cc-a — APPROVED** [review]
Great. Looks like a good summary.

**Pull request merged**